### DISCUSSION TITLE
Webradio with lots of FM music

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#479] [General]
[Synthwave Radio](https://www.synthwaveradio.eu/)

> We created this online synthwave radio station to inspire people how good synthwave music is. We also like to code and we wanted to share this amazing music genre called synthwave this way to you. We love how this music gives you oldschool 80s vibes and futuristic vibes at the same time. Our radio is streaming 24 hours a day, 7 days a week on 192kb/s and is Hosted in Finland. We like Synthwave, Vaporwave, Dreamwave, Retrowave and Synthpop. Our goal is to give underrated artists a stage on our radio. Thanks for your visit and enjoy the music!

Listening to this for a while should give everyone ideas what do do with our favorite little instrument :-)

**Comment by Banana71** [comment]
Wow, thank you for sharing this with us. I'm excited, exactly my thing. Very inspiring

**Reply by probonopd** [reply]
Just listened to "Marvel83' - Misty Horizons" on that radio station. That "gong", e.g., at 4:21. Most likely a DX7?
https://www.youtube.com/watch?v=1bArfuGF6QU

**Reply by Banana71** [reply]
I think you mean the song Stellar Wind by Marvel'83. There the typical bell sound that you can heard at 4:21. The song Misty Horizon is only 1:54 long.
I love the puristic synth sound in the song "Marvel'83 - The Year Of Dreamland" from 1:05. I suspect a Prophet 5 behind it. That would be another challenge to create this sound for the miniDexed.

**Reply by probonopd** [reply]
Well I mean the video I linked, actually. But thanks for the correction and the hints ;-)

**Comment by Banana71** [comment]
miniDexed PolySynth Demo: 
3 track recording, PolySynth, Drums and Pad. All Sounds from the miniDexed. PolySynth got a little bit Delay and Phaser

https://user-images.githubusercontent.com/104296205/232575054-17007a9f-69be-43ab-9e81-f079d9f37e3c.mp4

**Comment by probonopd** [comment]
More stations:

https://nightride.fm/stations

**Comment by Banana71** [comment]
@probonopd, Thanks to your post here I came across the band GUNSHIP. 
Since then it has been a constant representative in my playlists.
I really like Songs like Monster in Paradise, Ghost, The Drone Racing League, Tech Noir, ....

Art: GUNSHIP - Ghost:  https://youtu.be/aUJuwNxNUWQ?si=VTv4VCZ394Ju1avQ
Fun: GUNSHIP - The Drone Racing League:  https://youtu.be/3kOWOAi5_2o?si=iiLLvL1n-3oHxs3D
80s backflash: GUNSHIP - When You Grow Up, Your Heart Dies:  https://youtu.be/ri9IefTuNzc?si=OSHR53UL8JCIx_km

**Comment by plamikcho** [comment]
I like this radio too. One thing I wish to have in miniDexed - a good saxophone performance (not sure if this is possible). Something like this: https://youtu.be/14ynC2ScI18?t=152. I tried all available patches but with my limited knowledge and capabilities it is impossible for me to create such sounds.

**Reply by Banana71** [reply]
The saxophone is one of the most expressive instruments ever. Volume, modulation, different blowing techniques, growl, vibrato, ... there is a lot to consider. For a realistic saxophone emulation, many playing aids are necessary, aftertouch, modulation wheel, breath controller. Conversely, this also means that great skills are required of the keyboard player.

Yamaha's answer was: YAMAHA VL1, VL1-M, VL7, VL70-M PHYSICAL MODELING
https://www.youtube.com/watch?v=M0l1ekQ7hNQ

But I will try it
